src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/convert_reduce_no_keep_dims.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/common_utils.hpp"
+#include "common_test_utils/node_builders/reduce.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/general_utils.h"
+
+using namespace CPUTestUtils;
+using namespace ov::intel_cpu;
+
+namespace ov {
+namespace test {
+
+typedef std::tuple<int,                             // Axe to reduce order
+                   ov::test::utils::ReductionType,  // Reduce operation type
+                   std::vector<InputShape>          // Input shapes
+                   >
+    ReduceConvertCPUTestParamsSet;
+
+class ReduceNoKeepDimsTransformationCPUTest: public testing::WithParamInterface<ReduceConvertCPUTestParamsSet>,
```

Comment:
I think the idea was to write a unit test instead.
Since there is a corresponding single layer test which covers the inference functionality already.

Reply:
Replaced subgraph test with unit test